piman.py

```diff
@@ -114,7 +115,7 @@ def restart(switch_address, interface, ports):
 
 
 def reinstall(switch_address, interface, port):
-    with open("reinstall.txt", "w") as f:
+    with open(reinstall_file, "w+") as f:
         switch_ip_parts = switch_address.split(".")
         pi_addr = ".".join([switch_ip_parts[0], switch_ip_parts[1], str(private_number), str(port)])
```

Comment:
i really think we should change the arguments to include the full IP address. that way we don't need to do this formatting here. it also makes the incorrect assumption that the port number will always be the last octet.

if you want to leave it like this, that is ok.